prebid-mobile/modules/rendering/android-sdk-integration-max.md

```diff
@@ -0,0 +1,344 @@
+---
+layout: page_v2
+title: Google Ad Manager Integration
+description: Integration of Prebid Rendering module whith Google Ad Manager  
+sidebarType: 2
+---
+
+# AppLovin MAX Integration
+
+The integration of Prebid Mobile with AppLovin MAX assumes that publisher has MAX account and has already integrated the AppLovin MAX SDK into the app.
+
+See the [AppLovin MAX Documentation](https://dash.applovin.com/documentation/mediation/ios/getting-started/integration) for the MAX integration details.
```

Comment:
you have added the ios link here, we can link the android guide:
https://dash.applovin.com/documentation/mediation/android/getting-started/integration